src/style.css

```diff
@@ -1 +1,60 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  display: block;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+  position: relative;
+  bottom: 4px;
```

Comment:
let's update your code ) please check this article and make the correct CSS properties order 
https://netology-university.bitbucket.io/codestyle/css/#order